wormhole-connect/src/views/v2/Bridge/index.tsx

```diff
@@ -369,7 +369,7 @@ const Bridge = () => {
           ) {
             const route = routeStates.find((rs) => rs.name === selectedRoute);
 
-            if (route?.available) {
+            if (route?.supported && route?.available) {
```

Comment:
Although useAvailableRoutes won't set available prop if a route is not supported, I thought it won't hurt to add a check here as well.